### ISSUE TITLE
[BUG]  Documentation issue - Interpreting GUI

### ISSUE THREAD

**Issue by mm12** [sharppy/SHARPpy#238]
**Describe the bug**
The help documentation is missing information on what the color magenta means. 

**To Reproduce**
N/A: Go to [issue location](https://sharppy.github.io/SHARPpy/interpreting_gui.html#wind-speed-profile)

**Expected behavior**
There should be information on all current colors used

**Screenshots**
![image](https://user-images.githubusercontent.com/5874051/171062054-3bc3079e-ec32-437b-a6ab-09ca1218d213.png)
<img width="255" alt="image" src="https://user-images.githubusercontent.com/5874051/171062070-f7bf9046-534f-4bf2-b2c5-dfa61c3dc997.png">
<img width="229" alt="image" src="https://user-images.githubusercontent.com/5874051/171062084-c3280069-9a57-4e27-88eb-b1ea05f0e10f.png">

**Versioning Information (please complete the following information):**
N/A
 
** 

**Additional context**
Sorry for the low-quality report, but FR and BR templates both didn't seem right.

**Comment by keltonhalbert** [comment]
So the information you screenshot is technically correct - the folks at the College of DuPage changed the colors themselves on their own internal build. I don't think it would make sense for us to document all colors, including theirs, since it's non-standard and customized to behave so.  It also looks like they made the wind barb colors separate from the hodograph colors, and instead are colored by wind speed. 

In summary: Can't really support their use case in official documentation since its their custom version

**Comment by mm12** [comment]
Understandable, I suspected this might have been the case.

**Comment by mzuranski** [comment]
To answer this question from COD's side, and for any future internet travelers, the magenta color on both the hodograph and wind speed w/ height are for the 0-1 km layer.

You bring up a good point about documenting this on our page, I'll see what we can do though it may not be right away.
